PullRequestCreated/PullRequestCreated.ts

```diff
@@ -1,13 +1,19 @@
 import { OctokitAction } from '../lib/OctokitAction';
 import { PullRequest } from '../lib/OctokitTypes';
-import { JIRA_ISSUE_PATTERN, JIRA_TASK_ISSUE } from '../lib/Constants';
+import { JIRA_ISSUE_PATTERN } from '../lib/Constants';
+
+interface Issue {
+  type: string;
+  parent?: string;
+}
 
 class PullRequestCreated extends OctokitAction {
   protected async execute(): Promise<void> {
     const pr = await this.getPullRequest(this.payload.pull_request.number);
     if (this.shouldCreateIssue(pr)) {
-      const projectKey = this.getInput('jira-project');
-      const issueKey = await this.jira.createIssue(projectKey, JIRA_TASK_ISSUE, pr.title);
+      const issue = await this.newIssueTypeAndParent(pr);
+      const projectKey = this.getInput('jira-project', true);
+      const issueKey = await this.jira.createIssue(projectKey, issue.type, pr.title, { parent: { key: issue.parent } });
```

Comment:
`issueKey` can be null if something goes wrong. We should add `if(issueKey != null)` before updating the title